resources/nbd_loop_device/actions/run.yaml

```diff
@@ -0,0 +1,52 @@
+- hosts: [{{ host }}]
+  sudo: yes
+  tasks:
+    - name: install nbd-server
+      apt: name=nbd-server state=latest
+    - name: install nbd-client
+      apt: name=nbd-client state=latest
+    - name: prepare a file
+      command: truncate -s 10G {{path}} creates={{path}}
+    - name: check if loop for file is already created
+      shell: sudo losetup -a|grep {{path}}
+      register: loop_created
+      ignore_errors: True
+    - name: if loop is not created, create it
+      command: losetup -f {{path}}
+      when: loop_created|failed
+    - name: find loop device
+      shell: losetup -a|grep '{{path}}'|awk -F':' '{print $1}'
+      register: loop_device
+    - name: check if loop device has gpt
+      shell: "sudo parted {{ '{{' }} loop_device.stdout }} print | grep 'Partition Table: gpt'"
+      register: gpt_created
+      ignore_errors: True
+    - name: set gpt label
+      shell: "echo yes | sudo parted {{ '{{' }} loop_device.stdout }} mklabel gpt"
+      ignore_errors: True
+      when: gpt_created|failed
+    - name: check if guid set
+      shell: "sgdisk -i1 {{ '{{' }} loop_device.stdout }} | grep 'Partition GUID code: 4FBD7E29-9D25-41B8-AFD0-062C0CEFF05D'"
+      register: guid_set
+      ignore_errors: True
+    - name: set guid
+      shell: "sgdisk -n 1:0:9G -t 1:4FBD7E29-9D25-41B8-AFD0-062C0CEFF05D -p {{ '{{' }} loop_device.stdout }}"
+      ignore_errors: True
+      when: guid_set|failed
+    - name: Creates directory
+      file: path=/mnt/dev state=directory owner=nbd
+    - name: Create symlink to loop device
+      file: src={{ '{{' }} loop_device.stdout }} dest=/mnt{{ '{{' }} loop_device.stdout }} state=link
+    - name: is nbd served
+      shell: "netstat -ln | grep '127.0.0.1:4242 '"
+      register: nbd_served
+      ignore_errors: True
+    - name: configure loop as nbd
+      shell: nbd-server 127.0.0.1@4242 /mnt{{ '{{' }} loop_device.stdout }} -c
+      ignore_errors: True
+      when: nbd_served|failed
+    - name: Set owner for loop
+      shell: "chown nbd /mnt{{ '{{' }} loop_device.stdout }}"
+    - name: connect to nbd
+      shell: nbd-client 127.0.0.1 4242 /dev/nbd0
```

Comment:
You shouldn't hardcode /dev/nbd0 We may want to have more disks. It's better to add input in meta yaml like `device` and set default value to /dev/nbd0